src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -249,232 +214,242 @@ RCLConsensus::acquireTxSet(RCLTxSet::ID const & setId)
+    auto initialSet = std::make_shared<SHAMap>(
         SHAMapType::TRANSACTION, app_.family(), SHAMap::version{1});
-    initialSet->setUnbacked ();
+    initialSet->setUnbacked();
 
     // Build SHAMap containing all transactions in our open ledger
     for (auto const& tx : initialLedger->txs)
     {
-        Serializer s (2048);
+        Serializer s(2048);
         tx.first->add(s);
-        initialSet->addItem (
-            SHAMapItem (tx.first->getTransactionID(), std::move (s)), true, false);
+        initialSet->addItem(
+            SHAMapItem(tx.first->getTransactionID(), std::move(s)),
+            true,
+            false);
     }
 
     // Add pseudo-transactions to the set
-    if ((app_.config().standalone() || (proposing && correctLCL))
-            && ((prevLedger->info().seq % 256) == 0))
+    if ((app_.config().standalone() || (proposing && !wrongLCL)) &&
+        ((prevLedger->info().seq % 256) == 0))
     {
         // previous ledger was flag ledger, add pseudo-transactions
         auto const validations =
-            app_.getValidations().getValidations (
-                prevLedger->info().parentHash);
+            app_.getValidations().getValidations(prevLedger->info().parentHash);
 
-        std::size_t const count = std::count_if (
-            validations.begin(), validations.end(),
-            [](auto const& v)
-            {
+        std::size_t const count = std::count_if(
+            validations.begin(), validations.end(), [](auto const& v) {
                 return v.second->isTrusted();
             });
 
-        if (count >= app_.validators ().quorum ())
+        if (count >= app_.validators().quorum())
         {
-            feeVote_->doVoting (
-                prevLedger,
-                validations,
-                initialSet);
-            app_.getAmendmentTable ().doVoting (
-                prevLedger,
-                validations,
-                initialSet);
+            feeVote_->doVoting(prevLedger, validations, initialSet);
+            app_.getAmendmentTable().doVoting(
+                prevLedger, validations, initialSet);
         }
     }
 
     // Now we need an immutable snapshot
     initialSet = initialSet->snapShot(false);
     auto setHash = initialSet->getHash().as_uint256();
 
-    return std::make_pair<RCLTxSet, RCLCxPeerPos::Proposal> (
-        std::move (initialSet),
-        RCLCxPeerPos::Proposal {
-            initialLedger->info().parentHash,
-            RCLCxPeerPos::Proposal::seqJoin,
-            setHash,
-            closeTime,
-            now,
-            nodeID_ });
+    return Result(
+        std::move(initialSet),
+        RCLCxPeerPos::Proposal{initialLedger->info().parentHash,
+                               RCLCxPeerPos::Proposal::seqJoin,
+                               setHash,
+                               closeTime,
+                               app_.timeKeeper().closeTime(),
+                               nodeID_});
```

Comment:
@bachase Thanks. It looks like the brace initializers isn't handled like we want. If I replace that with parens I get: ```
return Result(
        std::move(initialSet),
        RCLCxPeerPos::Proposal(
            initialLedger->info().parentHash,
            RCLCxPeerPos::Proposal::seqJoin,
            setHash,
            closeTime,
            app_.timeKeeper().closeTime(),
            nodeID_));
```